src/ChainSafe.Gaming.HyperPlay/HyperPlayProvider.cs

```diff
@@ -78,6 +79,7 @@ public override async Task<string> Connect()
                 await localStorage.Save(data);
             }
 
+            await AddNetworkIfNotExistInWallet();
```

Comment:
this was already handled in EthereumWindow.jslib, it's not working?

Reply:
Oh it was? Well I've added it there since I removed if from the WalletSigners part 😬 I doubt it was added tbh there because adding the actual network did require more fields than we had previously in regards to the currency itself.

Reply:
it was adding when I tested it [here](https://github.com/ChainSafe/web3.unity/issues/1030)

Reply:
You were right Rob. My appologies